GLTFSDK/Source/MeshPrimitiveUtils.cpp

```diff
@@ -847,19 +650,18 @@ std::vector<float> MeshPrimitiveUtils::GetTexCoords_1(const Document& doc, const
 // Colors
 std::vector<uint32_t> MeshPrimitiveUtils::GetColors(const Document& doc, const GLTFResourceReader& reader, const Accessor& colorsAccessor)
```

Comment:
Two thoughts for GetColors:

1. Unsigned int formats supported by the core spec for this attribute shouldn't be getting converted to float and back. The original code would be preferable for these two component types.
2. What should happen for signed int data with negative values? I'm not sure that all possible component types make sense for some attributes.

Reply:
Fixed 1; 2 is part of the general error handling discussion so I'll resolve this for now.